config/initializers/ihat_supported_formats.rb

```diff
@@ -1,26 +1,2 @@
-Tahi::Application.config.ihat_supported_formats = nil
-module IhatSupportedFormats
-  def self.call
-    if ENV['IHAT_URL'].present?
-      begin
-        response = Faraday.get(ENV['IHAT_URL'])
-        if response && response.body
-          Tahi::Application.config.ihat_supported_formats =
-            JSON.dump(JSON.parse(response.body))
-        else
-          warn "Invalid JSON response from #{ENV['IHAT_URL']}"
-        end
-      rescue Faraday::ConnectionFailed
-        warn "Unable to connect to #{ENV['IHAT_URL']}"
-      end
-    else
-      warn "ENV['IHAT_URL'] Not set, falling back to default document types…"
-    end
-  end
-end
-
-def warn(message)
-  Rails.logger.warn message
-end
-
-IhatSupportedFormats.call unless Rails.env.test?
+Tahi::Application.config.ihat_supported_import_formats = %w(docx odt)
+Tahi::Application.config.ihat_supported_export_formats = %w(docx)
```

Comment:
We had several reasons for removing this check.  We have started to change our mindset a bit and treat the new iHat pipeline as a completely separate application.  It has its own feature timeline and used by applications other than Tahi.  We want Tahi (and not iHat) to be in control of when new "features" are released.   Severing the dependency between the two applications seems to achieve that purpose.  Since Tahi should soon start releasing on a routine basis, adding additional import/export formats and including it with a release seems like a low barrier.

Rather than hard code these values like we did here, it might actually be a better approach to use some kind of feature toggle or environment variables instead.  This would allow controls to turn off certain iHat import/export functionality if a bug or issue is found in iHat.